### PR TITLE
Fixed: Parent taxon selection on taxon creation

### LINKED ISSUE #11777: Allow to select parent taxon on creation
**Describe the proposed solution**
When I create a taxon I want to be able to select the parent taxon immediately instead of waiting until I have saved the taxon.

Before save:

image

After save:

image

### PR DESCRIPTION
**Pull request by igormukhingmailcom**
| Q               | A
| --------------- | -----
| Branch?         | 1.5
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #11777
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonType.php [hunk 1]
```diff
@@ -33,15 +33,9 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
...
-                }
-
-                $event->getForm()->add('parent', TaxonAutocompleteChoiceType::class, [
-                    'label' => 'sylius.form.taxon.parent',
-                    'required' => false,
-                ]);
+            ->add('parent', TaxonAutocompleteChoiceType::class, [
+                'label' => 'sylius.form.taxon.parent',
+                'required' => false,
             })
```
Comment (loevgaard) [inline src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonType.php:39]:
```suggestion
            ])
```

Comment (igormukhingmailcom) [reply src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonType.php]: Force pushed

**Comment by lchrusciel** [comment]
Hey Igor, thanks for the contribution. Despite red build (which is probably nor related to PR, but I've restarted build), can you add one scenario to our behat tests that will cover this feature? Let's make it land in 1.7 as well

**Comment by loevgaard** [comment]
A scenario where you select a parent on creation I guess? :)

**Comment by lchrusciel** [comment]
yes

**Comment by igormukhingmailcom** [comment]
Rebased to 1.7 and added behat test, @lchrusciel

**Comment by lchrusciel** [comment]
Thanks, Igor! :tada:

**Comment by loevgaard** [comment]
Thank you, Igor and Lukasz!

**Pull request merged**